processing/physical_range.py

```diff
@@ -312,93 +310,58 @@ def make_plot(self, x_data, y_data, var_obj, year, thresholds=True):
-        plt.ylim((y_min, y_max))
-        plt.subplot(*subplot_pos)
-        plt.title(title, fontsize=self.plot_config.plot_title_fontsize)
-        plot = plt.plot_date
-        plot(x_data, y_data, c='0.75',
-             ms=2, ls='', lw=0)
-        error_outliers, warning_outliers = self.identify_outliers(
-            x_data, y_data, var_obj)
+        ax.set_ylim((y_min, y_max))
+        ax.set_xlabel(f'Time ({y_name})', fontsize=self.plot_config.plot_title_fontsize)
+        ax.set_ylabel(f'{var_obj.name} ({var_obj.units})', fontsize=self.plot_config.plot_title_fontsize)
+        ax.set_title(title, fontsize=self.plot_config.plot_title_fontsize)
+        
+        ax.plot_date(x_data, y_data, color='0.75', ms=2, ls='', lw=0)
+        
+        error_outliers, warning_outliers = self.identify_outliers(x_data, y_data, var_obj)
         if error_outliers:
-            plot(error_outliers[0], error_outliers[1], 'o', markersize=6,
-                 markeredgewidth=1, markerfacecolor='None',
-                 markeredgecolor='r')
+            ax.plot_date(error_outliers[0], error_outliers[1], 'o', markersize=6, markeredgewidth=1, markerfacecolor='None', markeredgecolor='r')
         if warning_outliers:
-            plot(warning_outliers[0], warning_outliers[1], 'o', markersize=6,
-                 markeredgewidth=1, markerfacecolor='None',
-                 markeredgecolor='orange')
-        plt.axhline(y=var_obj.max_lim, marker='', linestyle='-',
-                    linewidth=1, color='orange')
-        plt.axhline(y=var_obj.min_lim, linestyle='-',
-                    linewidth=1, color='orange')
-        plt.axhline(y=var_obj.error_max, linestyle='-',
-                    linewidth=1, color='r')
-        plt.axhline(y=var_obj.error_min, linestyle='-',
-                    linewidth=1, color='r')
+            ax.plot_date(warning_outliers[0], warning_outliers[1], 'o', markersize=6, markeredgewidth=1, markerfacecolor='None', markeredgecolor='orange')
+        
+        ax.axhline(y=var_obj.max_lim, linestyle='-', linewidth=1, color='orange')
+        ax.axhline(y=var_obj.min_lim, linestyle='-', linewidth=1, color='orange')
+        ax.axhline(y=var_obj.error_max, linestyle='-', linewidth=1, color='r')
+        ax.axhline(y=var_obj.error_min, linestyle='-', linewidth=1, color='r')
 
     def plot(self, var_obj, year):
-        start, end = self.annual_idx[year]['start'], self.annual_idx[year][
-            'end']
+        start, end = self.annual_idx[year]['start'], self.annual_idx[year]['end']
         annual_data = self.data[var_obj.name][start:end + 1]
-        time_data = [self.ts_util.cast_as_datetime(t)
-                     for t in self.data['TIMESTAMP_START'][start:end + 1]]
-        fig = plt.figure(1, figsize=(12, 13))  # Create figure
+        time_data = [self.ts_util.cast_as_datetime(t) for t in self.data['TIMESTAMP_START'][start:end + 1]]
+        
+        fig, (ax1, ax2) = plt.subplots(2, 1, figsize=(12, 13))  # Create figure

```

Comment:
What are ax1 and ax2? The subplots?
Consider a different variable name that is a bit more human readable.